core/src/test/scripts/test_data/study_es_0/data_fusions.txt

```diff
@@ -1,4 +1,5 @@
 Hugo_Symbol	Entrez_Gene_Id	Center	Tumor_Sample_Barcode	Fusion	DNA_support	RNA_support	Method	Frame
 FGFR3	2261	saturn	TEST-A2B8-01	TMPRSS2-ERG fusion	yes	yes	unknown	unknown
 ERBB2	2064	jupiter	TEST-A2FF-01	Fusion2	yes	yes	unknown	unknown
-ERBB2	2064	uranus	TCGA-GI-A2C8-01	ERBB2-TEST Fusion	yes	yes	unknown	frameshift
\ No newline at end of file
+ERBB2	2064	uranus	TCGA-GI-A2C8-01	ERBB2-TEST Fusion	yes	yes	unknown	frameshift
+TP53	7157	uranus	TCGA-GI-A2C8-01	TP53-TEST Fusion	yes	yes	unknown	frameshift
```

Comment:
@n1zea144 can you please remove this one? In study_es_0 the idea is to have data that does *not* result in any warnings. You could add it to study_es_3, which is the study where we test the warnings.